### PR TITLE
Split interface into frontend and rest modules

### PR DESCRIPTION
**Pull request by jan-xyz**
This pull request targets issue #23.

It splits the api out of the main `picasso.py` and prepares for a better separation which allows a better overview and modularity in the future. Now, with this ground work it'll be possible to rewrite the front-end to use the REST api. 

The helper class is a by-product of the two interfaces being in the same module before. It provides logic that is currently needed with both. In the future I was planning to put the entire logic into a helper class (call it `backend_connector`) that is used by the REST API to provide its information and consumed by the front-end (I imagined this to be written in JavaScript).

### CODE REVIEW

**Comment by jan-xyz** [comment]
In issue #23 you are talking about having two separate files. Instead of refactoring the old web-ui I want to write it from scratch while already using REST to access the information from the backend. I want to get this pull-request through before starting on the next step to clarify if there are any structural issues with my approach. So, please give me feedback if the api refactoring works for you. I would implement it the same way (flask blueprints; naming convention; putting logic into a helper/backend connector).

**Comment by rhsimplex** [comment]
Hey, I generalized the settings a bit.  Can you rebase or merge master into your PR? should be pretty straightforward

File: picasso/helper.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+from picasso.visualizations.base import BaseVisualization
+import inspect
+from flask import (
+    g,
+    current_app
+)
+from picasso.models.base import load_model
+
+
+class Helper:
```
Comment (rhsimplex) [inline picasso/helper.py]: I like the idea of splitting these helper functions out into a separate file.  We don't need a new class here though (all the members are static!) -- can you just import the functions into `picasso.py` directly?  And rename `helper.py` to `utils.py` or preferably something even more descriptive.  "helpers" is a bit too generic.

**Comment by jan-xyz** [comment]
I updated the pull request with your suggestions. Good catch! :)

**Comment by rhsimplex** [comment]
Looks good, just made a mini PR with some formatting stuff.  Looks like no documentation needs updating, so this is ready to merge after the formatting updates.

**Comment by jan-xyz** [comment]
As always your final touches to my PR 👍 . Merged your changes so all we have to do is wait of the checks to pass.

**Comment by rhsimplex** [comment]
My test for cosmetic changes is this: if I'm too lazy to do it myself, it's not worth asking the requester!

**Pull request merged**